Source/WebKit/Platform/IPC/Connection.cpp

```diff
@@ -1263,7 +1263,7 @@ void Connection::enqueueIncomingMessage(UniqueRef<Decoder> incomingMessage)
             return;
 
         if (isIncomingMessagesThrottlingEnabled() && m_incomingMessages.size() >= maxPendingIncomingMessagesKillingThreshold) {
-            if (kill()) {
+            if (m_client && m_client->requestRemoteProcessTermination()) {
```

Comment:
Is this called on the main thread?

Reply:
Oh, that is a good point! I will check this and add an assert.

Thanks for reviewing!